CI fails building stm32u5  tests/subsys/pm/device_runtime_api

**Describe the bug**
Building tests/subsys/pm/device_runtime_api on b_u585i_iot02a  or nucleo_u575zi_q platforms fails due to 
drivers/serial/uart_stm32.c:1699: undefined reference to `LL_EXTI_EnableIT_0_31'

Missing `#include <stm32_ll_exti.h>`

Please also mention any information which could help others to understand
the problem you're facing:
 - this is a regression : https://github.com/zephyrproject-rtos/zephyr/pull/50100


**To Reproduce**
Steps to reproduce the behavior:
1. west build -p auto -b b_u585i_iot02a tests/subsys/pm/device_runtime_api3. make
2. building error 

**Expected behavior**
build success
**Environment (please complete the following information):**
 - OS: ubuntu
 - Toolchain SDK 0.15.0
 - Commit SHA 9105f1ef61961142271c764feae94a4b335c75d9

--- a/drivers/serial/uart_stm32.c
+++ b/drivers/serial/uart_stm32.c
@@ -35,6 +35,9 @@
 
 #include <stm32_ll_usart.h>
 #include <stm32_ll_lpuart.h>
+#if defined(CONFIG_PM) && defined(IS_UART_WAKEUP_FROMSTOP_INSTANCE)
+#include <stm32_ll_exti.h>
+#endif /* CONFIG_PM */
 
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(uart_stm32, CONFIG_UART_LOG_LEVEL);

